ui/components/multichain/ramps-card/index.scss

```diff
@@ -4,15 +4,15 @@
   &__cta-button {
     width: fit-content;
     color: var(--brand-colors-grey-grey800);
-    background-color: var(--brand-colors-white-white000);
+    background-color: var(--brand-colors-white);
   }
 
   &__title {
-    color: var(--brand-colors-white-white000);
+    color: var(--brand-colors-white);
   }
 
   &__body {
-    color: var(--brand-colors-white-white000);
+    color: var(--brand-colors-white);
     width: 80%;
```

Comment:
### After
Ramps card text and button remain white

<img width="1512" alt="Screenshot 2024-06-10 at 3 15 58 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/47c1daa0-0e4d-48a5-bd18-50670cca299c">
<img width="1500" alt="Screenshot 2024-06-10 at 3 16 19 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/ec28215d-a4d6-420a-8ffe-d9bf5a08563e">